### PR TITLE
docs: device-preparation

### PR DESCRIPTION
**Pull request by KazuCocoa**
I have confirmed this setting allows me to automate web content in chrome with:

```
server_url = 'http://localhost:4723'
caps = {
  "platformName": "ios",
  "automationName": "xcuitest",
  "udid": "00008020-000E5CDA0A23002E",
  "bundleId": "com.google.chrome.ios",
  "webDriverAgentUrl": "http://192.168.4.255:8100"
}
core = Appium::Core.for url: server_url, caps: caps
driver = core.start_driver
driver.execute_script 'mobile:getContexts'
```

### CODE REVIEW

File: docs/device-preparation.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Adva
 Make sure the destination `WKWebView` and/or `JSContext` component have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) property set to `true`.
 Read [Enabling the Inspection of Web Content in Apps](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) for more details on this property.
 
+#### Chrome v115+ and iOS 16.4+ support Web testing availability on Chrome
+
+Please turn on _Web Inspector_ on iOS device via Chrome app -> _Settings_ -> _Content Settings_ -> _Web Inspector_ -> Turn _Web Inspector_ on -> Kill the Chrome app process
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:37]: >  Kill the Chrome app process

Is it the setting name?

Comment (KazuCocoa) [reply docs/device-preparation.md]: no, it is via app switcher or app history. I have modified a bit, whats about now?

File: docs/device-preparation.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Adva
 Make sure the destination `WKWebView` and/or `JSContext` component have [`isInspectable`](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) property set to `true`.
 Read [Enabling the Inspection of Web Content in Apps](https://webkit.org/blog/13936/enabling-the-inspection-of-web-content-in-apps/) for more details on this property.
 
+#### Chrome v115+ and iOS 16.4+ support Web testing availability on Chrome
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:35]: "Chrome" is mentioned twice

File: docs/device-preparation.md [hunk 3]
```diff
@@ -32,6 +32,14 @@ Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Adva
...
+
+As `isInspectable` is available, Chrome started providing the availability in the release package.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:37]: Chrome browser for iOS now provides a remote debugging feature for release versions of the app after the `isInspectable` property has been introduced by Apple.

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**